### PR TITLE
README change for WSDL to datasource

### PR DESCRIPTION
**Pull request by rashmihunt**
This PR contains README changes for using SOAP WebService datasource as input instead of WSDL as input.
@crandmck @raymondfeng PTAL

### CODE REVIEW

**Comment by crandmck** [comment]
@rashmihunt what's the best way for me to install the CLI with this update, so I can try/test it myself?

**Comment by crandmck** [comment]
Since I cannot try this out, I have no way of testing it, so I can't really test the doc.

File: soap/README_Soap.md [hunk 1]
```diff
@@ -49,22 +49,45 @@ select 'empty-server' for this demo purpose.
...
 
-### Generate APIs from WSDL
+### Create a SOAP Web Services datasource
 
-Now let's try to generate APIs from WSDL document. Note: There is an enhancement planned to generate 
-APIs given SOAP Web Service data source.
+The next step is to create SOAP Web Services datasource. In this demo, we will create a SOAP datasource
+for an externally available Periodic Table Web Service: http://www.webservicex.net/periodictable.asmx?WSDL                                                       
+                                                         
+
```
Comment (crandmck) [inline soap/README_Soap.md]: Need to move `cd soap-demo` from line 58 to just above this cmd.

**Comment by rashmihunt** [comment]
@crandmck Fixed README review comments. PTAL

**Pull request merged**